src/brasil/gov/agenda/profiles/testing/metadata.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
```

Comment:
Since the default language in brasil.gov.agenda is portuguese, we changed the language in tests to be portuguese, as well with the robot tests.